operations/agent-flow-mixin/alerts/clustering.libsonnet

```
@@ -25,5 +25,50 @@ alert.newGroup(
       'Cluster nodes have entered a split brain state.',
       '5m',
     ),
+
+    // Standard Deviation of Lamport clock time between nodes is too high
+    alert.newRule(
+      'ClusterLamportClockDrift',
+      'stddev by (cluster, namespace) (cluster_node_lamport_time) != 0',
+      "Cluster nodes' lamport clocks are not converging.",
+      '5m'
+    ),
+
+    // Nodes health score is not zero.
+    alert.newRule(
+      'ClusterNodeUnhealthy',
+      |||
+        sum by (cluster, namespace, instance) (cluster_node_gossip_health_score) > 0
```

There's nothing to aggregate here, since there's just one of these metrics per node, and we want to track it at the node level.

```suggestion
        cluster_node_gossip_health_score > 0
```